dbms/src/TableFunctions/TableFunctionS3.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <TableFunctions/ITableFunctionFileLike.h>
+#include <Interpreters/Context.h>
```

Comment:
Fwd declaration is enough.

Reply:
c03857b2aee08fee4616cecae6caf4dbd94555bf